docs/current/sdk/nodejs/snippets/get-started/step3/build.js

```diff
@@ -4,7 +4,7 @@ import { connect } from "@dagger.io/dagger"
 connect(async (client) => {
   // get Node image
   // get Node version
-  const node = client.container().from("node:16").withExec(["node", "-v"])
+  const node = client.container().from("node:16").withExec({ args: ["node", "-v"] })
```

Comment:
withExec is taking a requirement args as a array of strings
`withExec(args: string[], opts?: ContainerWithExecOpts)`
No need to use an object ;). The original code is correct.